__tests__/main.test.ts

```diff
@@ -99,6 +100,26 @@ describe("VIB", () => {
       expect(executionGraph).toBeDefined()
       expect(executionGraph["status"]).toEqual("SUCCEEDED")
     }, 1200000) // long test, processing this execution graph ( lint, trivy ) might take up to 2 minutes.
+
+    it("Runs the GitHub action and fails because of a timeout", async () => {
+      jest.spyOn(core, "setFailed")
+      const startTime = Date.now()
+      const pipelineDuration = getNumberInput((process.env["INPUT_MAX-PIPELINE-DURATION"] = "200"))
+      const executionGraph = await runAction()
+      const config = await loadConfig()
+      const executionGraphId = await createPipeline(config)
+      fixedExecutionGraphId = executionGraph["execution_graph_id"]
+      for (const task of executionGraph["tasks"]) {
+        if (task["action_id"] === "trivy") {
+          fixedTaskId = task["task_id"]
+        }
+      }
+
+      if (Date.now() - startTime > pipelineDuration) {
```

Comment:
This check should not be done. The `expect` clauses inside are ok, but the `if` condition should not exist